EduNLP/I2V/i2v.py

```diff
@@ -61,8 +62,8 @@ def __init__(self, tokenizer, t2v, *args, tokenizer_kwargs: dict = None, pretrai
         if tokenizer == 'bert':
             self.tokenizer = BertTokenizer(**tokenizer_kwargs if tokenizer_kwargs is not None else {})
```

Comment:
Here we need to use `BertTokenizer.from_pretrained()` for `I2V.Bert`

Reply:
use BertTokenizer.from_pretrained() now